app/api/events.py

```diff
@@ -31,18 +34,63 @@ class Meta:
 
     id = fields.Str(dump_only=True)
     identifier = fields.Str(dump_only=True)
-    name = fields.Str()
-    event_url = fields.Url()
-    starts_at = fields.DateTime(required=True)
-    ends_at = fields.DateTime(required=True)
-    tickets = Relationship(attribute='ticket',
+    name = fields.Str(required=True)
+    email = fields.Str(required=True)
```

Comment:
@shubham-padia 
what about the `/get` method , it still shows `InvalidRequestError: Entity '<class 'app.models.role.Role'>' has no property 'deleted_at'`